src/elements/content-sidebar/Sidebar.js

```diff
@@ -133,9 +130,12 @@ class Sidebar extends React.Component<Props, State> {
      */
 
     handleAnnotationSelect = (annotationId: string | null): void => {
```

Comment:
Should this move into ActivitySidebar? It seems odd for it to live in this file while other aspects live further down.

Reply:
Thoughts on this?

Reply:
I think we can leave it for now, because moving it up would probably require moving `handleVersionHistoryClick` up as well. We could address this in a separate PR